src/transactions.js

```diff
@@ -26,18 +26,35 @@ let release = null;
 (() => {
   try {
     const raw = readFileSync(TXFILE, "utf-8");
-    const txidState = JSON.parse(raw);
-    currentTransactionId = txidState.txid;
+
+    const entries = raw
+      .split("\n")
+      .slice(0, -1)
```

Comment:
What are you doing here, removing a final newline? Why is there one in a file we write 😅 ?